### PR TITLE
Add constant for MICROSOFT_AZURE_OVERRIDE_CONTAINER_PATH 

### PR DESCRIPTION
**Pull request by rickalee**
MICROSOFT_AZURE_OVERRIDE_CONTAINER_PATH - Override Container name in the Image URL

### CODE REVIEW

**Comment by rickalee** [comment]
@hugosolar Can you test this? This was requested by Systems and added by Shakir.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ For multisites or to enforce Azure Blob Storage settings, you can define the fol
 * MICROSOFT_AZURE_CONTAINER - Azure Blob Container
 * MICROSOFT_AZURE_CNAME - Domain: must start with http(s)://
 * MICROSOFT_AZURE_USE_FOR_DEFAULT_UPLOAD - boolean (default false)
-
+* MICROSOFT_AZURE_OVERRIDE_CONTAINER_PATH - Override Container name in the Image URL , can be just "/"
```
Comment (jeffpaul) [inline README.md:33]: @rickalee probably also want to add into `readme.txt` as well?

**Review by hugosolar — DISMISSED** [review]
tested and work as expected

**Comment by rickalee** [comment]
@jeffpaul Updated readme.txt

**Pull request merged**